spec/factories/statements.rb

```diff
@@ -4,8 +4,8 @@
     year { Faker::Number.between(from: 2021, to: 2024) }
     deadline_date { Faker::Date.forward(days: 30) }
     payment_date { Faker::Date.forward(days: 30) }
-    cohort { association :cohort }
-    lead_provider { association :lead_provider }
+    cohort
```

Comment:
I think we should default to `cohort { create(:cohort, :current) }` otherwise some validations can break when a random year is picked from the future.